### ISSUE TITLE
Using just “Greek” for the new "el-polyton" is ambiguous

### ISSUE THREAD

**Issue by yvanzo** [houseabsolute/DateTime-Locale#37]
[`DateTime::Locale::Catalog` 1.38](https://metacpan.org/release/DROLSKY/DateTime-Locale-1.38/view/lib/DateTime/Locale/Catalog.pm) is currently naming “Greek” both `el` and `el-polyton`, whereas it only really fits the first code `el`. The new `el-polyton` (to CLDR 43) seems to stand for “Polytonic Greek” according to <https://en.wikipedia.org/wiki/Greek_diacritics#Computer_encoding>.
```csv
Locale code      Locale name (in English)                    Native locale name
==========================================================================================================
el               Greek                                             Ελληνικά
el-CY            Greek Cyprus                                      Ελληνικά Κύπρος
el-GR            Greek Greece                                      Ελληνικά Ελλάδα
el-polyton       Greek                                             Ἑλληνικά
```

It handles country codes correctly, but not the polytonic variant.

It seems that the name of this variant is available upstream: https://github.com/unicode-org/cldr-json/blob/43.0.0/cldr-json/cldr-localenames-full/main/en/variants.json#L51

**Comment by autarch** [comment]
Thanks for the bug report. This is fixed in the v1.39 release.

**Comment by yvanzo** [comment]
Thank you!
